transcom/mymove
# B 20709 int

## [B-20709](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A983171)

## Summary

This task introduces the lockedPriceCents field to MS and CS service items. The field is meant to store the price of these service items at the time the move is made available to the prime. The field has also been added to the API so that it is returned to the front end in preparation for follow-on tasks.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office application
2. Login as a TOO
3. Find a move that does not have an approved shipment
4. Approve the first shipment with at least a move management fee or counseling fee. Recommended to do both if the move supports it.
5. Validate the the locked_price_cents column in the database is populated for the new MS and CS service items you created.
6. Open the debugger and refresh the page
7. Verify the locked_price_cents is returned in the API when requesting service items

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
